addons/web/static/src/scss/search_view.scss

```diff
@@ -82,7 +82,6 @@
                     }
                     &.o-expand, &.o-expanded { // caret
                         @include o-position-absolute($left: 6px);
-                        padding: 3px;
```

Comment:
is this ok for both community AND enterprise?

Reply:
@aab-odoo tested on enterprise branch also and UI looks fine after this